modules/drep_chunked.nf

```diff
@@ -0,0 +1,50 @@
+/*
+ * dRep, this process dereplicates a set of genomes.
+ * this version of the process is meant to be use for a subset
+ * of the genomes (as part of the drep_swf for large catalogues)
+*/
+process DREP_CHUNKED {
```

Comment:
It looks like `drep.nf` and `drep_chunked.nf` are very similar and can be combined into one process.

Reply:
They are, I didn't merge them to avoid some extra work and because they are very likely to differ in the near future. My suggestion is for us to keep both copies for the time being.